config/vufind/FeedbackForms.yaml

```diff
@@ -176,3 +189,76 @@ forms:
         options:
           - Yes
           - No
+
+
+  # Form configuration for unit testing
```

Comment:
Rather than putting test configurations in the real configuration file, it would be better to mock the YamlReader in the unit test so that it returns this configuration. If you need help doing that, let me know and I should be able to take care of it.

Reply:
The mocked YamlReader now reads the test data from fixtures.

Reply:
Thanks!